### PR TITLE
feat: dynamically elapse time left in cal video sidebar

### PR DESCRIPTION
**Pull request by G3root**
## What does this PR do?

increments time left in cal video sidebar dynamically rather than calculate on Mount

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: apps/web/pages/video/[uid].tsx [hunk 1]
```diff
@@ -280,7 +313,10 @@ export async function getServerSideProps(context: GetServerSidePropsContext) {
       ...(typeof bookingObj.references[0].meetingPassword === "string" && {
         meetingPassword: bookingObj.references[0].meetingPassword,
       }),
-      booking: bookingObj,
+      booking: {
+        ...bookingObj,
+        ...(bookingObj.description && { description: md.render(bookingObj.description) }),
```
Comment (G3root) [inline apps/web/pages/video/[uid].tsx:318]: render markdown in serverside rather than the client side

Comment (emrysal) [reply apps/web/pages/video/[uid].tsx:318]: @PeerRich A booking description is not a smart text editor, why are we rendering it in markdown here? 🤔

Comment (PeerRich) [reply apps/web/pages/video/[uid].tsx:318]: no clue

Comment (PeerRich) [reply apps/web/pages/video/[uid].tsx:318]: it is markdown

**Review by PeerRich — APPROVED** [review]

**Pull request merged**